src/Api/Controllers/OrganizationsController.cs

```diff
@@ -366,10 +366,12 @@ public async Task<ProfileOrganizationResponseModel> PostSubscribeSecretsManagerA
             throw new NotFoundException();
         }
 
-        var userId = _userService.GetProperUserId(User).Value;
+        var updateOrganization = await _addSecretsManagerSubscriptionCommand.SignUpAsync(organization, model.AdditionalSmSeats,
+            model.AdditionalServiceAccounts);
```

Comment:
You don't need to return any value from `SignUpAsync` anymore. You already have the orgId to make the repo call below.